floor/floor/controller/midi_test.py

```diff
@@ -92,17 +100,14 @@ def test_midi_passthru(self):
         self.inject_control_mode_change(21, 127)
         self.assertEqual(1, self.controller.playlist.stop_playlist.call_count)
 
-        self.assertEqual(4, self.controller.processor.handle_midi_command.call_count)
+        self.assertEqual(0, self.controller.processor.handle_midi_command.call_count)
+        self.inject_control_mode_change(48, 99)
+        self.assertEqual(1, self.controller.processor.handle_midi_command.call_count)
         self.controller.processor.handle_midi_command.assert_has_calls([
-            mock.call(('note_on', 'C3', 127)),
-            mock.call(('note_on', 'C4', 55)),
-            mock.call(('control_mode_change', 21, 11)),
-            mock.call(('control_mode_change', 21, 127)),
+            mock.call(self.controller.processor.__class__, 2, 99),
         ])
 
     def test_note_name_translation(self):
+        self.assertEqual(0, self.controller.playlist.stay.call_count)
         self.inject_note_on('Csn1', 127)
-        self.assertEqual(1, self.controller.processor.handle_midi_command.call_count)
-        self.controller.processor.handle_midi_command.assert_called_once_with(
-            ('note_on', 'C#-1', 127)
-        )
```

Comment:
The note name is not being passed through now, so I think its enough to check that injecting this as `Csn1` calls a function registered as `C#-1`